File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):

    return eval(binomial)

# 다른 풀이
a, op, b = binomial.split()
if op == '+': return int(a) + int(b)
if op == '-': return int(a) - int(b)
if op == '*': return int(a) * int(b)

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {

        String[] Arr = binomial.split(" ");
        int a = Integer.parseInt(Arr[0]);
        int b = Integer.parseInt(Arr[2]);
        
        return switch(Arr[1]) {
            case "+" -> a + b;
            case "-" -> a - b;
            case "*" -> a * b;
            default -> 0;
        };
    }
}